about.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About me</title>
+  <link href="styles/style.css" rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Merriweather&family=Montserrat&family=Sacramento&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <nav>
+      <ul class="nav_links">
+        <li>
+          <a href="index.html">Home</a>
+        </li>
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a href="portfolio.html">Portfolio</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+
+
+
+  <main>
+    <div class="project-container">
+
+      <div class="title">
+        <h2>My skills</h2>
+      </div>
+      <div class="project-image">
+        <img src="images/skills.png" alt="skills">
```

Comment:
The alt text for images should be more descriptive. People who are blind or have low vision use screen readers to tell them what is on a webpage. Screen readers read the alt text to describe the picture to the user. This picture called "skills" will not explain to the user what is in the picture.